test/sqatt/src/Sqatt.hs

```diff
@@ -301,11 +302,11 @@ runTxsWithExample mLogDir ex = Concurrently $ do
       putStr "."
     timer = Concurrently $ do
       sleep sqattTimeout
-      throwIO TestTimedOut
+      return $ Left TestTimedOut
     txsUIProc mUiLogDir imf port =
-      Concurrently $ try $ do
+      Concurrently $ do
```

Comment:
Why was the `try` removed? We still need to catch errors in case the `txsUIShell` aborts with an error (for instance if a port number was busy).

Reply:
It's back 👍